src/components/AnnotationViewer.vue

```diff
@@ -858,6 +985,14 @@ export default class AnnotationViewer extends Vue {
         const annotation = this.selectedTool.values.annotation;
         this.annotationLayer.mode(annotation?.shape);
         break;
+      case "snap":
+        if (this.selectedTool.values.snapTo.value === "circleToDot") {
```

Comment:
Isn't it an enum or something like that to ensure "circleToDot" typo?

Reply:
It corresponds to a value written in `public\config\templates.json` (line 46) and I can't think of a way of doing things better
In the rest of the code source these values are also checked against simple strings